src/Admin/GalleryAdmin.php

```diff
@@ -22,7 +22,7 @@
 use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
 
 /**
- * @phpstan-extends AbstractAdmin<\Sonata\MediaBundle\Model\GalleryInterface>
+ * @phpstan-extends AbstractAdmin<\Sonata\MediaBundle\Model\GalleryInterface<\Sonata\MediaBundle\Model\GalleryItemInterface>>
```

Comment:
Why on the media admin you added the template but not here? It looks strange

Reply:
And then the MediaAdmin is loosing the template from the BaseMedia so I would say it's kinda useless to make the BaseMedia generic

Reply:
I've removed the generic in `MediaAdmin`, as I think we can make that move in a different PR if it is required later.
Thank you for pointing.